app/models/participant_declaration.rb

```diff
@@ -5,13 +5,16 @@ class ParticipantDeclaration < ApplicationRecord
   belongs_to :cpd_lead_provider
   belongs_to :user
   belongs_to :participant_profile
+  belongs_to :superseded_by, class_name: "ParticipantDeclaration", optional: true
+  has_many :duplicate_participant_declarations, class_name: "ParticipantDeclaration", foreign_key: :superseded_by_id, inverse_of: :superseded_by
```

Comment:
What do you think about this association name? Does this read better than something like `supersedes`?